libraries/AP_HAL_ChibiOS/hwdef/F4BY_F427/defaults.parm

```diff
@@ -0,0 +1,7 @@
+BATT_AMP_OFFSET 2.316

```

Comment:
Replace this with a line like this in your hwdef:
```
define AP_BATT_CURR_AMP_OFFSET_DEFAULT 2.316
```

See `AP_BattMonitor_Analog.cpp` for the other default-define names.

Reply:
> See `AP_BattMonitor_Analog.cpp` for the other default-define names.

Thank you, Peter